client/app/tests/phpunit/Pact/SiriusApiGatewayClientTest.php

```diff
@@ -54,7 +51,12 @@ public function setUp(): void
     {
         $client = new GuzzleClient();
         $baseUrl = getenv('PACT_MOCK_SERVER_HOST');
-        $serializer = (self::bootKernel(['debug' => false]))->getContainer()->get('serializer');
+        $pactHostPort = getenv('PACT_MOCK_SERVER_PORT');
```

Comment:
Excluding the port means that these tests can't be run from the command line manually, as the port is effectively hard-coded to 80, and we run the pact-mock from docker-compose on port 1234.